README.rst

```diff
@@ -116,6 +119,52 @@ Usage
       latest     Get the latest tag in a changelog.
       tag        Create an annotated git tag based on a changelog entry.
       update     Updates the version and date of the most recent changelog entry.
+      create     Creates a new CHANGELOG.md file with a standard header.
+      entry      Add a quick entry to the current version.
+      next       Create a new unreleased version in changelog file.
+
+next
+----
+
+Create a new unreleased version in changelog file.
+
+::
+
+   Usage: chag next
+
+   Creates a new unreleased version inside of the changelog file.
+
+   Options:
+     --help     Displays this message.
+
+entry
```

Comment:
I'm not sure the entry command is a good idea long term. The description is right that this is mostly just useful for quick entries to be added to the changelog; trying to add things like multi-line entries would not be easy.  I would much rather prefer people to be verbose and descriptive when editing their changelogs. I think most people will be editing their code when they are getting ready to add an entry to the changelog. They are typically in an editor of some kind, and I think that's the more appropriate place to edit the changelog. Looking at the way it's implemented, I'm also wary of taking on the complexity of supporting this for such a (IMO) limited use case.